### PR TITLE
Adding verify_tls parameter to disable ssl/tls verification

### PR DESCRIPTION
**Pull request by rurod**
Hello,

Adding the verify_tls parameter to give the possibility to query prometheus without checking it's certificate.

### CODE REVIEW

**Comment by mattiascockburn** [comment]
This is super useful for testing and for circumstances where the use of a custom TLS store is not an option.

**Comment by rurod** [comment]
Resolved conflicts so this can be merged. Still waiting review.

**Comment by mattiascockburn** [comment]
@Lawouach would you mind looking at this? It does not change defaults and makes using the probe itself a bit more flexible.

File: chaosprometheus/probes.py [hunk 1]
```diff
@@ -23,6 +23,7 @@ def query(
     timeout: float = None,
     configuration: Configuration = None,
     secrets: Secrets = None,
+    verify_tls: bool = True,
```
Comment (Lawouach) [inline chaosprometheus/probes.py]: Could you put this before the `configuration` parameter? I tend, by convention, to keep the configuration and secrets as the last two arguments.

**Review by Lawouach — CHANGES_REQUESTED** [review]
Looks fine to me.

Can you had a changelog entry please?

**Comment by rurod** [comment]
Hi @Lawouach, done, hope its ok now.

**Review by Lawouach — APPROVED** [review]
LGTM

**Comment by Lawouach** [comment]
Thank you @rurod and getting back to it after so long.

**Comment by Lawouach** [comment]
I'll fix the linting after the merge. Thanks again!

**Pull request merged**